rclpy/rclpy/type_description_service.py

```diff
@@ -40,6 +40,7 @@ class TypeDescriptionService:
     def __init__(self, node):
         """Initialize the service, if the parameter is set to true."""
         node_name = node.get_name()
+        self._node = node
```

Comment:
So if we are setting `self._node` here so we can use it later in `_service_callback`, then presumably we don't need the `node` parameter to `_start_service`, right?  We can just use `self._node` there as well, I think?

Reply:
That's right